backend/app/utils/error_handlers.py

```
@@ -0,0 +1,490 @@
+        operation: Description of the operation that failed
+        context: Optional context information
+        request_id: Optional request correlation ID
+
+    Returns:
+        HTTPException with standardized error response
+    """
+    request_id = request_id or generate_request_id()
+
+    details = [
+        ErrorDetail(
+            message=f"Operation failed: {str(error)}",
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)